.gitignore

```diff
@@ -7,7 +7,6 @@
 .*.sw*
 .*.un~
 clover.xml
-composer.lock
```

Comment:
But that composer. lock is not the provided. Its the generated by composer update